input/mount-acquisition/event.json

```diff
@@ -1,6 +1,33 @@
 {
   "name": "event",
   "contents": [
+    // Core
     1,3
+    // Mount License
+    20,21,22,23,24,25,26,27,28,29,31,32,33,30,34,35,36,48,37,38,39,40,41,42,43,44,45,47,49,50,
```

Comment:
Even with your redefinition of 'event' as bound loot, I don't think any of these mount skins fit that categorization. I would say 'gem' in the first instance. Although I guess really I should add the different mount licenses at their own acquisition methods.

Reply:
They're both "gem" and "bound loot", since they can drop from RNG licenses. The original idea was to give them a different filter, "mount license" or something like that, but it was kinda pointless so they got the same treatment as the other accountbound RNG stuff. This distinction is there to know what's a direct gem store purchase vs a license purchase.

Reply:
Honestly it is all just gem store garbage to me :) But individual mount licenses do act like little guaranteed wardrobe unlocks so decent argument for going the whole hog there. Of course originally they were categorised by licence which covered this too, but it is more correct as acquisition method.

Reply:
Well, mounts have been outdated for a few years now, so I'd leave them like this for the moment. I had them split by Common/Canthan/Exclusive before, but that was kind of a mess, that's why I went back to the original.

Who knows, might find a better third way down the road lmao :D.